SygicMapsKit/Classes/BrowseMapModule/SYMKBrowseMapViewController.swift

```diff
@@ -216,13 +231,20 @@ public class SYMKBrowseMapViewController: SYMKModuleViewController {
         
     }
     
-    private func addCustomMarkersToMap() {
-        guard let markers = customMarkers, let mapController = mapController else { return }
+    private func addCustomMarkersToMap(_ markers: [SYMKMapPin]?) {
+        guard let markers = markers, markers.count > 0, let mapController = mapController else { return }
```

Comment:
ta `.count > 0` podmienka ani tu ani nizsie je zbytocna celkom, ale je to jedno v podstate